_posts/2021-05-04-how-to-update-your-ref-arch.adoc

```diff
@@ -112,9 +112,9 @@ context). As such, you need to inject these required blocks to use advanced feat
 root `terragrunt.hcl` configuration that includes these directives so that we can continue to use the original,
 non-Service Catalog based live configuration.
 
-For reference, you can download the https://github.com/gruntwork-io/terraform-aws-service-catalog/releases/tag/tbd[refarch-folder-structure TODO] zip file which
-contains all the following files in their respective locations in the folder structure. You'll have to copy these
-files into your own `infrastructure-live` repository.
+For reference, you can download the link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/releases/tag/v0.0.0-alpha1[refarch-folder-structure]
```

Comment:
Do you plan on updating this to the actual release tag eventually? Only reason I am asking is because this link currently is a 404.

Reply:
Yes. Please see https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/releases for the draft.

Reply:
Done in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/427